tests/test_playbooks/job_invocation.yml

```diff
@@ -0,0 +1,34 @@
+# registered host expected
```

Comment:
Your recorded fixtures contain an APIdoc dump, because we have a bug in the logic that throws it away.

Could you add the following to the beginning of the test file? It should get rid of the huge API doc by pre-warming the cache.

```yaml
---
- hosts: localhost
  collections:
    - theforeman.foreman
  gather_facts: false
  vars_files:
    - vars/server.yml
  tasks:
    - name: fetch status
      include_tasks: tasks/status_info.yml
```